learn/tanssi/index.md

```diff
@@ -1,8 +1,21 @@
 ---
 title: Learn About Tanssi
 description: Learn about Tanssi protocol and how it handles infrastructure complexities, making it easy for new appchains to be deployed in the Polkadot Web3 ecosystem.
-template: subsection-index-page.html
-hide: 
- - toc
- - feedback
+icon: octicons-code-16
+template: index-page.html
 ---
+
+# Learn About Tanssi
+
+Here, you’ll find everything that sets Tanssi apart as a leading solution for appchain development. From block production as a service to flexible security providers and essential integrations, this section highlights the core features that make Tanssi a robust and adaptable protocol for building scalable, secure, and innovative appchains.
```

Comment:
```suggestion
Here, you'll find an overview of Tanssi’s core capabilities that support appchain development, covering everything from infrastructure and security to essential integrations.
```

Since the below list includes block production as a service, flexible security providers, etc. lets keep this first sentence short and sweet